sdk/react/src/lib/fragment-components/a-popover.tsx

```diff
@@ -0,0 +1,282 @@
+			onChange={(_: any, { checked }: any) => setComponent({
+				...selectedComponent,
+				isShowCaret: checked
+		})} />
+
+        <Checkbox
+			labelText='Drop shadow'
+			id='drop-shadow'
+			checked={selectedComponent.isDropShadow}
+			onChange={(_: any, { checked }: any) => setComponent({
+				...selectedComponent,
+				isDropShadow: checked
+		})} />
+
+        <Checkbox
+			labelText='High contrast'
+			id='high-contrast'
+			checked={selectedComponent.isHighContrast}
+			onChange={(_: any, { checked }: any) => setComponent({
+				...selectedComponent,
+				isHighContrast: checked
+		})} />
+
+        <Dropdown
+			id='align'
+			label='Align'
+			titleText='Align'
+			items={alignItems}
+			selectedItem={alignItems.find(item => item.id === selectedComponent.align)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				align: event.selectedItem.id
+			})} />
+
+        <TextInput
+			value={selectedComponent.text}
+			labelText='Popover text'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				text: event.currentTarget.value
+			})} />
+    </>
+}
+
+export const APopOverCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+    value={selectedComponent.codeContext?.name}
+    labelText='Input name'
+    onChange={(event: any) => {
+        setComponent({
+            ...selectedComponent,
+            codeContext: {
+                ...selectedComponent.codeContext,
+                name: event.currentTarget.value
+            }
+        });
+    }} />;
+
+export const APopOver = ({
+	children,
+	componentObj,
+	...rest
+}: any) => {
+    return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+            <Popover 
+                align={componentObj.align}
+                caret={componentObj.isShowCaret}
+                dropShadow={componentObj.isDropShadow}
+                highContrast={componentObj.isHighContrast}
+                open={componentObj.isOpen}
+                className={cx(
+                    componentObj.cssClasses?.map((cc: any) => cc.id).join(' '),
+                    css`${styleObjectToString(componentObj.style)}`
+                )}>
+                    <svg _ngcontent-pla-c12="" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 32 32">
+                        <path _ngcontent-pla-c12="" d="M26,4H6A2,2,0,0,0,4,6V26a2,2,0,0,0,2,2H26a2,2,0,0,0,2-2V6A2,2,0,0,0,26,4ZM6,26V6H26V26Z">
+                        </path>
+                    </svg>
+                    <PopoverContent>
+                        {componentObj.text}
+                        {children}
+                    </PopoverContent>
+                </Popover>
+		</AComponent>
+    );
+}
+
+export const componentInfo: ComponentInfo = {
+	component: APopOver,
+	settingsUI: APopOverSettingsUI,
+	codeUI: APopOverCodeUI,
```

Comment:
Should be APopover, fix all instances,
```suggestion
	component: APopover,
	settingsUI: APopoverSettingsUI,
	codeUI: APopoverCodeUI,
```